pyscf/lib/dft/libxc_itrf.c

```diff
@@ -383,6 +383,27 @@ double LIBXC_hybrid_coeff(int xc_id)
         return factor;
 }
 
+void LIBXC_nlc_coeff(int xc_id, double *nlc_pars) {
+
+        xc_func_type func;
+        if(xc_func_init(&func, xc_id, XC_UNPOLARIZED) != 0){
+                fprintf(stderr, "XC functional %d not found\n", xc_id);
+                exit(1);
+        }
+        XC(nlc_coef)(&func, &nlc_pars[0], &nlc_pars[1]);
```

Comment:
I think this should also check if the functional actually is NLC.

Reply:
It seemed like LibXC was returning zeros if those parameters weren't defined so I thought this was okay. I might change it.